performance_tests/src/aws_encryption_sdk_performance_tests/master_key_providers/caching_cmm.py

```diff
@@ -0,0 +1,85 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Performance tests for the Caching Cryptographic Materials Manager (CMM)."""
+
+import aws_encryption_sdk
+
+
+def create_cmm(
+    kms_key_id: str,
+    max_age_in_cache: float,
+    cache_capacity: int
+):
+    """Demonstrate how to create a Caching CMM.
+
+    Usage: create_cmm(kms_key_id, max_age_in_cache, cache_capacity)
+    :param kms_key_id: Amazon Resource Name (ARN) of the KMS customer master key
+    :type kms_key_id: str
+    :param max_age_in_cache: Maximum time in seconds that a cached entry can be used
+    :type max_age_in_cache: float
+    :param cache_capacity: Maximum number of entries to retain in cache at once
+    :type cache_capacity: int
+    """
+    # Security thresholds
+    #   Max messages (or max bytes per) data key are optional
+    max_entry_messages = 100
```

Comment:
```suggestion
    max_messages_encrypted = 100
```
I'm not sure what `max_entry_messages` is supposed to mean. Just name it the variable name you're assigning it to

Reply:
I saw this in the example: https://github.com/aws/aws-encryption-sdk-python/blob/ebbc26b1fb83d7e0bcba66c70012730350b6c60d/examples/src/legacy/data_key_caching_basic.py#L20

Updated now,